### PR TITLE
Bug reporting and logs sharing improvements

### LINKED ISSUE #12042: Error reports improvement
# Feature Issue

We are receiving a lot of empty user error reports without any details, so will be nice to ask users for feedback in more user-friendly format to receive meaningful feedback.

### Description

Currently on shaking phone dialog is shown and on tapping "Report a bug" long template is shown.
photo_2021-04-26 14 30 09
Let's ask users at least for some feedback and paste it into description.
Also IMO template is too complicated 

```
1. Issue Description
(Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)


2. Steps to reproduce
(Describe how we can replicate the bug step by step.)
-Open Status
-...
-Step 3, etc.


3. Expected behavior
(Describe what you expected to happen.)


4. Actual behavior
(Describe what actually happened.)


5. Attach screenshots that can demo the problem, please
```

### Acceptance Criteria
Simplified version of user report inside app.
We can add fields, i.e.:
- description
- steps (not required)

And after that open mail client with pre-filled info.
Would appreciate some thoughts from UX and design side (@hesterbruikman @errorists )

### PR DESCRIPTION
**Pull request by rasom**
Changes:
- if logs archive can't be created (nil log level in release version) logs won't be attached at all
- in order to submit a new report, the user has to fill description/steps form where description is required. That is to avoid sending empty reports.
- on iOS logs archive can be shared without sending email at all

fix https://github.com/status-im/status-react/issues/12042

status: wip

### CODE REVIEW

**Comment by churik** [comment]
@rasom 
### ISSUE 1: fields are overlapped with keyboard on Android
IMAGE 2021-11-08 11:32:53

### ISSUE 2: If I closed error reporting form one time, can't open it again on shaking until app reopening (Android)

### ISSUE 3: On `Submit a bug` option from Need help (used messenger) - got `Archived logs` message (Android)
IMAGE 2021-11-08 11:58:46

### ISSUE 4:  can't create multiaccount on IOS
Got stuck on "Create password" screen after confirming password

IMAGE 2021-11-08 12:04:24
(couldn't reproduce on nightly build)

**Comment by rasom** [comment]
@churik 2-4 are fixed

1 is also fixed by using modal instead of bottom sheet, it just works there

**Comment by churik** [comment]
awesome work @rasom! 
Only one small improvement: can we close submitting form after redirect to mailclient (after tapping "Submit" button)? Looks a bit odd that after submission should close it manually.

**Comment by rasom** [comment]
@churik fixed

**Comment by churik** [comment]
@rasom 
last change introduced issue on IOS that after submitting from mailclient with the draft is not opened

https://user-images.githubusercontent.com/4557972/141099013-7587eb0e-8271-41e3-9b2c-03627ca29f5e.mp4

Please, mind my commit when squashing. thanks!

**Comment by churik** [comment]
@rasom
thank you!
I think we can go with https://github.com/status-im/status-react/pull/12773#issuecomment-964352153
But I suppose better to clean {{description}} and {{steps}} after form submission to prevent users to submit for again with empty values

https://user-images.githubusercontent.com/4557972/141127009-a64d1059-ec82-4866-a9f2-45199c9ae51c.mp4

**Comment by rasom** [comment]
@churik done

**Review by bitgamma — APPROVED** [review]

File: src/status_im/ui/screens/bug_report.cljs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  (:require [quo.core :as quo]
+            [quo.react-native :as react-native]
+            [re-frame.core :as re-frame]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.topbar :as topbar]))
+
+(defn bug-report []
+  (let [{:keys [description steps]} @(re-frame/subscribe [:bug-report/details])]
+    [react-native/view {:style {:flex 1}}
+     [topbar/topbar
```
Comment (flexsurfer) [inline src/status_im/ui/screens/bug_report.cljs:11]: better to use native topbar in this case

File: src/status_im/ui/screens/bug_report.cljs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      {:title  (i18n/label :t/bug-report)
+       :modal? true}]
+     [react-native/view {:style {:flex               1
```
Comment (flexsurfer) [inline src/status_im/ui/screens/bug_report.cljs:14]: this is something new, usually we use `react/view` or `rn/view`

File: translations/tl.json [hunk 3]
```diff
@@ -1194,7 +1194,7 @@
     "remove-token": "Alisin ang token",
     "removed": "alisin",
     "repeat-pin": "Ulitin ang bagong 6-digit na passcode",
-    "report-bug-email-template": "1. Paglalarawan ng Isyu\n(Ilarawan ang tampok na nais mo, o maikli ang buod ng bug at kung ano ang iyong ginawa, kung ano ang inaasahan mong mangyari, at kung ano ang tunay na mangyayari. Mga seksyon sa ibaba)\n\n\n2. Mga hakbang upang makalikha\n(Ilarawan kung paano namin maaaring kopyahin ang sunud-sunod na hakbang.)\n-Buksan ang Status\n-...\n-Step 3, atbp.\n\n3. Inaasahang pag-uugali\n(Ilarawan kung ano ang inaasahan mong mangyari.)\n\n\n4. Aktwal na pag-uugali\n(Ilarawan kung ano talaga ang nangyari.)\n\n\n5. Ikabit ang mga screenshot na maaaring mag-demo sa problema, kung maaari sana",
+    "report-bug-email-template": "1. Paglalarawan ng Isyu\{{description}}\n\n\n2. Mga hakbang upang makalikha\n {{steps}}\n\n\n3. Ikabit ang mga screenshot na maaaring mag-demo sa problema, kung maaari sana",
```
Comment (flexsurfer) [inline translations/tl.json:1197]: `n` is missed

**Pull request merged**